### ISSUE TITLE
Backup via Davx5 to Nextcloud 25.0.2 fails

### ISSUE THREAD

**Issue by t-m-w** [stevesoltys/backup#490]
Backup to latest Nextcloud server is not working via Davx5 (or the Nextcloud app, but that isn't new):
- Nextcloud server 23.0.11: Works
- Nextcloud server 24.0.8: Works
- Nextcloud server 25.0.2: Fails (backup failed notification)

It seems like something changed in Nextcloud server, which is now returning error 500 sometimes.

Unsure if this is related to other issues.

Took log with "Verbose logging" enabled for Davx5, and redacted many *very* sensitive portions.

Some notable log lines:
```
12-30 11:54:40.300 10165   791  1968 V davx5   : [HttpClient] <-- 500 https://nextcloud-server.lan/remote.php/dav/files/nextcloud-user/nextcloud-path/.SeedVaultAndroidBackup/1670000000000 (107ms)
[...]
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: Error finding file blocking
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: java.io.IOException: kotlinx.coroutines.TimeoutCancellationException: Timed out waiting for 15000 ms
[...]
12-30 11:54:55.322 10165   791  5695 E JavaBinder: *** Uncaught remote exception!  (Exceptions are not yet supported across processes.)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: java.lang.RuntimeException: at.bitfire.dav4jvm.exception.HttpException: HTTP 500 
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.os.Parcel.writeException(Parcel.java:2796)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.database.DatabaseUtils.writeExceptionToParcel(DatabaseUtils.java:117)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.content.ContentProviderNative.onTransact(ContentProviderNative.java:425)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.os.Binder.execTransactInternal(Binder.java:1280)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.os.Binder.execTransact(Binder.java:1244)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: Caused by: at.bitfire.dav4jvm.exception.HttpException: HTTP 500 
[...]
12-30 11:54:55.324 10143  5177  5203 E KVBackup: Error uploading DB
12-30 11:54:55.324 10143  5177  5203 E KVBackup: java.io.IOException
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at com.stevesoltys.seedvault.plugins.saf.DocumentsStorageKt.createOrGetFile(DocumentsStorage.kt:163)
```

<details><summary>More log</summary>

```
12-30 11:54:40.128 10165   791  1968 D davx5   : [settings.SettingsManager] Looking up setting proxy_type
12-30 11:54:40.129 10165   791  1968 D davx5   : [settings.SettingsManager] SharedPreferencesProvider: proxy_type = null
12-30 11:54:40.129 10165   791  1968 D davx5   : [settings.SettingsManager] DefaultsProvider: proxy_type = -1
12-30 11:54:40.130 10165   791  1968 D davx5   : [settings.SettingsManager] Looked up setting proxy_type -> -1
12-30 11:54:40.132 10165   791  1968 D davx5   : [settings.SettingsManager] Looking up setting distrust_system_certs
12-30 11:54:40.133 10165   791  1968 D davx5   : [settings.SettingsManager] SharedPreferencesProvider: distrust_system_certs = false
12-30 11:54:40.133 10165   791  1968 D davx5   : [settings.SettingsManager] Looked up setting distrust_system_certs -> false
12-30 11:54:40.138 10165   791  1968 D TrafficStats: tagSocket(101) with statsTag=0xffffffff, statsUid=-1
12-30 11:54:40.187 10165   791  1968 V davx5   : [HttpClient] --> PROPFIND https://nextcloud-server.lan/remote.php/dav/files/nextcloud-user/nextcloud-path/.SeedVaultAndroidBackup/1670000000000 h2
12-30 11:54:40.188 10165   791  1968 V davx5   : [HttpClient] Depth: 1
12-30 11:54:40.189 10165   791  1968 V davx5   : [HttpClient] User-Agent: DAVx5/4.2.6-ose (2022/12/12; dav4jvm; okhttp/4.10.0) Android/13
12-30 11:54:40.189 10165   791  1968 V davx5   : [HttpClient] Accept-Language: en-US, en;q=0.7, *;q=0.5
12-30 11:54:40.190 10165   791  1968 V davx5   : [HttpClient] Content-Type: application/xml; charset=utf-8
12-30 11:54:40.190 10165   791  1968 V davx5   : [HttpClient] Content-Length: 335
12-30 11:54:40.191 10165   791  1968 V davx5   : [HttpClient] Host: nextcloud-server.lan
12-30 11:54:40.191 10165   791  1968 V davx5   : [HttpClient] Connection: Keep-Alive
12-30 11:54:40.191 10165   791  1968 V davx5   : [HttpClient] Accept-Encoding: gzip
12-30 11:54:40.192 10165   791  1968 V davx5   : [HttpClient] Cookie: oc_sessionPassphrase=REDACTED; REDACTED; __Host-nc_sameSiteCookiestrict=true; __Host-nc_sameSiteCookielax=true
12-30 11:54:40.192 10165   791  1968 V davx5   : [HttpClient] --> END PROPFIND
12-30 11:54:40.192 10165   791  1968 D dav4jvm : [at.bitfire.dav4jvm.BasicDigestAuthHandler] Trying Basic auth preemptively
12-30 11:54:40.192 10165   791  1968 D dav4jvm : [at.bitfire.dav4jvm.BasicDigestAuthHandler] Adding Basic authorization header for https://nextcloud-server.lan/remote.php/dav/files/nextcloud-user/nextcloud-path/.SeedVaultAndroidBackup/1670000000000
12-30 11:54:40.235  1000  1491  1491 I sysui_multi_action: [757,1501,758,1,793,203,794,0,795,203,796,1,806,com.stevesoltys.seedvault,857,NotificationBackupObserver,858,2,947,0,1500,0,1688,1]
12-30 11:54:40.235  1000  1491  1491 I sysui_multi_action: [757,804,799,note_interruptive,801,0,802,1]
12-30 11:54:40.241  1000  1491  1491 W NotificationHistory: Attempted to add notif for locked/gone/disabled user 0
12-30 11:54:40.260  1000  1491  1693 I input_interaction: Interaction with: f02adf3 StatusBar (server), [Gesture Monitor] swipe-up (server), [Gesture Monitor] edge-swipe (server), [Gesture Monitor] onehanded-touch (server), PointerEventDispatcher0 (server), 
12-30 11:54:40.265 10135  1816  1816 I sysui_multi_action: [757,803,799,panel_open,802,1]
12-30 11:54:40.265 10135  1816  1816 I sysui_multi_action: [757,1328,758,4,1326,66,1327,0,1329,0]
12-30 11:54:40.300 10165   791  1968 V davx5   : [HttpClient] <-- 500 https://nextcloud-server.lan/remote.php/dav/files/nextcloud-user/nextcloud-path/.SeedVaultAndroidBackup/1670000000000 (107ms)
12-30 11:54:40.301 10165   791  1968 V davx5   : [HttpClient] server: nginx
12-30 11:54:40.301 10165   791  1968 V davx5   : [HttpClient] date: Fri, 30 Dec 2022 16:54:38 GMT
12-30 11:54:40.302 10165   791  1968 V davx5   : [HttpClient] content-type: text/html; charset=UTF-8
12-30 11:54:40.302 10165   791  1968 V davx5   : [HttpClient] content-length: 0
12-30 11:54:40.302 10165   791  1968 V davx5   : [HttpClient] referrer-policy: no-referrer
12-30 11:54:40.302 10165   791  1968 V davx5   : [HttpClient] x-content-type-options: nosniff
12-30 11:54:40.303 10165   791  1968 V davx5   : [HttpClient] x-frame-options: SAMEORIGIN
12-30 11:54:40.303 10165   791  1968 V davx5   : [HttpClient] x-permitted-cross-domain-policies: none
12-30 11:54:40.303 10165   791  1968 V davx5   : [HttpClient] x-robots-tag: none
12-30 11:54:40.304 10165   791  1968 V davx5   : [HttpClient] x-xss-protection: 1; mode=block
12-30 11:54:40.304 10165   791  1968 V davx5   : [HttpClient] x-powered-by: PHP/8.1.13
12-30 11:54:40.304 10165   791  1968 V davx5   : [HttpClient] expires: Thu, 19 Nov 1981 08:52:00 GMT
12-30 11:54:40.304 10165   791  1968 V davx5   : [HttpClient] cache-control: no-store, no-cache, must-revalidate
12-30 11:54:40.305 10165   791  1968 V davx5   : [HttpClient] pragma: no-cache
12-30 11:54:40.305 10165   791  1968 V davx5   : [HttpClient] content-security-policy: default-src 'none';
12-30 11:54:40.305 10165   791  1968 V davx5   : [HttpClient] set-cookie: REDACTED; path=/; secure; HttpOnly; SameSite=Lax
12-30 11:54:40.306 10165   791  1968 V davx5   : [HttpClient] set-cookie: REDACTED2; path=/; secure; HttpOnly; SameSite=Lax
12-30 11:54:40.306 10165   791  1968 V davx5   : [HttpClient] <-- END HTTP
[...]
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: Error finding file blocking
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: java.io.IOException: kotlinx.coroutines.TimeoutCancellationException: Timed out waiting for 15000 ms
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at com.stevesoltys.seedvault.plugins.saf.DocumentsStorageKt.listFilesBlocking(DocumentsStorage.kt:210)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at com.stevesoltys.seedvault.plugins.saf.DocumentsStorageKt$listFilesBlocking$1.invokeSuspend(Unknown Source:13)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.internal.ScopeCoroutine.afterResume(Scopes.kt:33)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:102)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:274)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:1)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:1)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at com.stevesoltys.seedvault.transport.ConfigurableBackupTransport.finishBackup(ConfigurableBackupTransport.kt:136)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at android.app.backup.BackupTransport$TransportImpl.finishBackup(BackupTransport.java:775)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at com.android.internal.backup.IBackupTransport$Stub.onTransact(IBackupTransport.java:670)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at android.os.Binder.execTransactInternal(Binder.java:1285)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at android.os.Binder.execTransact(Binder.java:1244)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: Caused by: kotlinx.coroutines.TimeoutCancellationException: Timed out waiting for 15000 ms
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.TimeoutKt.TimeoutCancellationException(Timeout.kt:186)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.TimeoutCoroutine.run(Timeout.kt:156)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.EventLoopImplBase$DelayedRunnableTask.run(EventLoop.common.kt:497)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:274)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at kotlinx.coroutines.DefaultExecutor.run(DefaultExecutor.kt:69)
12-30 11:54:55.111 10143  5177  5203 E DocumentsStorage: 	at java.lang.Thread.run(Thread.java:1012)
12-30 11:54:55.114 10165   791  5695 D davx5   : [webdav.DavDocumentsProvider] WebDAV isChildDocument 4 590
12-30 11:54:55.117 10165   791  5695 D davx5   : [webdav.DavDocumentsProvider] WebDAV createDocument 590 application/octet-stream a-seedvault-subfolder
12-30 11:54:55.119 10165   791  5695 D davx5   : [settings.SettingsManager] Looking up setting proxy_type
12-30 11:54:55.120 10165   791  5695 D davx5   : [settings.SettingsManager] SharedPreferencesProvider: proxy_type = null
12-30 11:54:55.121 10165   791  5695 D davx5   : [settings.SettingsManager] DefaultsProvider: proxy_type = -1
12-30 11:54:55.122 10165   791  5695 D davx5   : [settings.SettingsManager] Looked up setting proxy_type -> -1
12-30 11:54:55.126 10165   791  5695 D davx5   : [settings.SettingsManager] Looking up setting distrust_system_certs
12-30 11:54:55.127 10165   791  5695 D davx5   : [settings.SettingsManager] SharedPreferencesProvider: distrust_system_certs = false
12-30 11:54:55.128 10165   791  5695 D davx5   : [settings.SettingsManager] Looked up setting distrust_system_certs -> false
12-30 11:54:55.137  1000  1491  7489 W ProcessStats: Tracking association SourceState{c5c41d0 system/1000 BTopFgs #217348} whose proc state 2 is better than process ProcessState{77bec6a nodomain.freeyourgadget.gadgetbridge/10325 pkg=nodomain.freeyourgadget.gadgetbridge} proc state 3 (13 skipped)
12-30 11:54:55.140 10165   791  5695 D TrafficStats: tagSocket(104) with statsTag=0xffffffff, statsUid=-1
12-30 11:54:55.213 10165   791  5695 V davx5   : [HttpClient] --> PUT https://nextcloud-server.lan/remote.php/dav/files/nextcloud-user/nextcloud-path/.SeedVaultAndroidBackup/1670000000000/a-seedvault-subfolder h2
12-30 11:54:55.214 10165   791  5695 V davx5   : [HttpClient] If-None-Match: *
12-30 11:54:55.214 10165   791  5695 V davx5   : [HttpClient] User-Agent: DAVx5/4.2.6-ose (2022/12/12; dav4jvm; okhttp/4.10.0) Android/13
12-30 11:54:55.215 10165   791  5695 V davx5   : [HttpClient] Accept-Language: en-US, en;q=0.7, *;q=0.5
12-30 11:54:55.217 10165   791  5695 V davx5   : [HttpClient] Content-Length: 0
12-30 11:54:55.217 10165   791  5695 V davx5   : [HttpClient] Host: nextcloud-server.lan
12-30 11:54:55.218 10165   791  5695 V davx5   : [HttpClient] Connection: Keep-Alive
12-30 11:54:55.219 10165   791  5695 V davx5   : [HttpClient] Accept-Encoding: gzip
12-30 11:54:55.219 10165   791  5695 V davx5   : [HttpClient] Cookie: oc_sessionPassphrase=REDACTED; REDACTED2; __Host-nc_sameSiteCookiestrict=true; __Host-nc_sameSiteCookielax=true
12-30 11:54:55.220 10165   791  5695 V davx5   : [HttpClient] --> END PUT
12-30 11:54:55.220 10165   791  5695 D dav4jvm : [at.bitfire.dav4jvm.BasicDigestAuthHandler] Trying Basic auth preemptively
12-30 11:54:55.221 10165   791  5695 D dav4jvm : [at.bitfire.dav4jvm.BasicDigestAuthHandler] Adding Basic authorization header for https://nextcloud-server.lan/remote.php/dav/files/nextcloud-user/nextcloud-path/.SeedVaultAndroidBackup/1670000000000/a-seedvault-subfolder
12-30 11:54:55.284 10135  1816  1873 D LocalImageResolver: Couldn't use ImageDecoder for drawable, falling back to non-resized load.
12-30 11:54:55.306 10165   791  5695 V davx5   : [HttpClient] <-- 500 https://nextcloud-server.lan/remote.php/dav/files/nextcloud-user/nextcloud-path/.SeedVaultAndroidBackup/1670000000000/a-seedvault-subfolder (85ms)
12-30 11:54:55.307 10165   791  5695 V davx5   : [HttpClient] server: nginx
12-30 11:54:55.308 10165   791  5695 V davx5   : [HttpClient] date: Fri, 30 Dec 2022 16:54:53 GMT
12-30 11:54:55.308 10135  1816  1873 D LocalImageResolver: Couldn't use ImageDecoder for drawable, falling back to non-resized load.
12-30 11:54:55.309 10165   791  5695 V davx5   : [HttpClient] content-type: text/html; charset=UTF-8
12-30 11:54:55.310 10165   791  5695 V davx5   : [HttpClient] content-length: 0
12-30 11:54:55.310 10165   791  5695 V davx5   : [HttpClient] referrer-policy: no-referrer
12-30 11:54:55.311 10165   791  5695 V davx5   : [HttpClient] x-content-type-options: nosniff
12-30 11:54:55.312 10165   791  5695 V davx5   : [HttpClient] x-frame-options: SAMEORIGIN
12-30 11:54:55.313 10165   791  5695 V davx5   : [HttpClient] x-permitted-cross-domain-policies: none
12-30 11:54:55.313 10165   791  5695 V davx5   : [HttpClient] x-robots-tag: none
12-30 11:54:55.314 10165   791  5695 V davx5   : [HttpClient] x-xss-protection: 1; mode=block
12-30 11:54:55.314 10165   791  5695 V davx5   : [HttpClient] x-powered-by: PHP/8.1.13
12-30 11:54:55.315 10165   791  5695 V davx5   : [HttpClient] expires: Thu, 19 Nov 1981 08:52:00 GMT
12-30 11:54:55.316 10165   791  5695 V davx5   : [HttpClient] cache-control: no-store, no-cache, must-revalidate
12-30 11:54:55.316 10165   791  5695 V davx5   : [HttpClient] pragma: no-cache
12-30 11:54:55.317 10165   791  5695 V davx5   : [HttpClient] content-security-policy: default-src 'none';
12-30 11:54:55.317 10165   791  5695 V davx5   : [HttpClient] set-cookie: REDACTED; path=/; secure; HttpOnly; SameSite=Lax
12-30 11:54:55.318 10165   791  5695 V davx5   : [HttpClient] set-cookie: REDACTED2; path=/; secure; HttpOnly; SameSite=Lax
12-30 11:54:55.318 10165   791  5695 V davx5   : [HttpClient] <-- END HTTP
12-30 11:54:55.322 10165   791  5695 E JavaBinder: *** Uncaught remote exception!  (Exceptions are not yet supported across processes.)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: java.lang.RuntimeException: at.bitfire.dav4jvm.exception.HttpException: HTTP 500 
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.os.Parcel.writeException(Parcel.java:2796)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.database.DatabaseUtils.writeExceptionToParcel(DatabaseUtils.java:117)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.content.ContentProviderNative.onTransact(ContentProviderNative.java:425)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.os.Binder.execTransactInternal(Binder.java:1280)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.os.Binder.execTransact(Binder.java:1244)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: Caused by: at.bitfire.dav4jvm.exception.HttpException: HTTP 500 
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at at.bitfire.dav4jvm.DavResource.checkStatus(DavResource.kt:5)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at at.bitfire.dav4jvm.DavResource.checkStatus(DavResource.kt:3)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at at.bitfire.dav4jvm.DavResource.put(DavResource.kt:26)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at at.bitfire.dav4jvm.DavResource.put$default(DavResource.kt:29)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at at.bitfire.davdroid.webdav.DavDocumentsProvider.createDocument(DavDocumentsProvider.kt:179)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.provider.DocumentsProvider.callUnchecked(DocumentsProvider.java:1139)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.provider.DocumentsProvider.call(DocumentsProvider.java:1078)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.content.ContentProvider.call(ContentProvider.java:2511)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.content.ContentProvider$Transport.call(ContentProvider.java:525)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	at android.content.ContentProviderNative.onTransact(ContentProviderNative.java:295)
12-30 11:54:55.322 10165   791  5695 E JavaBinder: 	... 2 more
12-30 11:54:55.324 10143  5177  5203 E KVBackup: Error uploading DB
12-30 11:54:55.324 10143  5177  5203 E KVBackup: java.io.IOException
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at com.stevesoltys.seedvault.plugins.saf.DocumentsStorageKt.createOrGetFile(DocumentsStorage.kt:163)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at com.stevesoltys.seedvault.plugins.saf.DocumentsStorageKt$createOrGetFile$1.invokeSuspend(Unknown Source:13)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at kotlinx.coroutines.internal.ScopeCoroutine.afterResume(Scopes.kt:33)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:102)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:274)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:1)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:1)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at com.stevesoltys.seedvault.transport.ConfigurableBackupTransport.finishBackup(ConfigurableBackupTransport.kt:136)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at android.app.backup.BackupTransport$TransportImpl.finishBackup(BackupTransport.java:775)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at com.android.internal.backup.IBackupTransport$Stub.onTransact(IBackupTransport.java:670)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at android.os.Binder.execTransactInternal(Binder.java:1285)
12-30 11:54:55.324 10143  5177  5203 E KVBackup: 	at android.os.Binder.execTransact(Binder.java:1244)
```
</details>

**Comment by chirayudesai** [comment]
Does trying to copy a bunch of files (perhaps even a SeedVault internal storage backup) work?

**Comment by t-m-w** [comment]
tl;dr: **Disable Nextcloud's Circles app.** See the information here: nextcloud/server#32432

<br/>

Primary problem: Reproducible failures during the backup process:
- Nextcloud server log entry: `Error | PHP | Nesting level too deep - recursive dependency? at /var/www/html/lib/private/Log/ExceptionSerializer.php#215`
- Viewing Davx5 mount contents will intermittently show error 500 when refreshing, and in logs (at least if verbose logging is enabled for Davx5).
- **Resolution: Disable Nextcloud's Circles app. See nextcloud/server#32432.**

<br/>

Secondary problem: Intermittent but immediate failures *starting* the backup process due to file locking problems:
- Nextcloud server log entry: `Error | no app in context | OCA\DAV\Connector\Sabre\Exception\FileLocked: "nextcloud-user/.SeedVaultAndroidBackup/1670000000000/.backup.metadata" is locked`
- Resolution: Unknown.
- Workaround: Try backing up again a little later, and it should work.

**Comment by t-m-w** [comment]
Upstream issue with Nextcloud server. Workaround noted above. Closing.
